### PR TITLE
fix /api/licenses/<id> again

### PR DESCRIPTION
**Pull request by opqdonut**
- there were two conflicting License schemas
- the single license GET api wasn't tested

part of #852
caused by #1064
see also #1061

### CODE REVIEW

File: test/clj/rems/api/test_licenses.clj [hunk 1]
```diff
@@ -37,21 +37,21 @@
...
-                         read-body)
-                license (->> body
-                             (filter #(= (:title %) (:title command)))
-                             first)]
+          (let [license (-> (request :get (str "/api/licenses/" id))
+                            (authenticate api-key user-id)
+                            handler
+                            assert-response-is-ok
+                            read-body)]
+            (is id)
```
Comment (luontola) [inline test/clj/rems/api/test_licenses.clj]: `(is id)` is testing the create operation, so it should be move to line 47½.

File: test/clj/rems/api/test_licenses.clj [hunk 2]
```diff
@@ -37,21 +37,21 @@
...
-            (json-body command)
-            handler
-            assert-response-is-ok)
+                                          :attachment-id nil}}}
+            id (-> (request :post "/api/licenses/create")
+                   (authenticate api-key user-id)
+                   (json-body command)
+                   handler
+                   assert-response-is-ok
+                   read-body
```
Comment (luontola) [inline test/clj/rems/api/test_licenses.clj]: There is a helper `read-ok-body`

File: test/clj/rems/api/test_licenses.clj [hunk 3]
```diff
@@ -65,21 +65,20 @@
...
+                   :id)]
```
Comment (luontola) [inline test/clj/rems/api/test_licenses.clj]: Add `(is id)` here

**Comment by opqdonut** [comment]
also added similar tests for the indivirual resource and license endpoints

**Pull request merged**